sonaranalyzer-dotnet/src/SonarAnalyzer.Utilities/Rules.Description/S4035.html

```diff
@@ -4,7 +4,7 @@
 <p>Alternatively <code>IEqualityComparer&lt;T&gt;</code> provides a safer interface and is used by collections or <code>Equals</code> could be made
 <code>virtual</code>.</p>
 <p>This rule raises an issue when a unsealed, <code>public</code> or <code>protected</code> class implements <code>IEquitable&lt;T&gt;</code> and the
-<code>Equals</code> is not <code>virtual</code>.</p>
+<code>Equals</code> is neither <code>virtual</code> nor <code>abstract</code>.</p>
```

Comment:
running `rspec cs S4035` should also modify `sonaranalyzer-dotnet/rspec/cs/S4035_c#.html`

Reply:
Indeed, for some reason `rspec` script is modifying more files, so I had to select them manually and I missed `S4035_c#.html`. Thanks!

Reply:
I opened #2572 to track this